src/python/grpcio/grpc/_server.py

```diff
@@ -278,6 +278,12 @@ def set_trailing_metadata(self, trailing_metadata):
             self._state.trailing_metadata = trailing_metadata
 
     def abort(self, code, details):
+        # treat OK like other invalid arguments: fail the RPC
+        if code == grpc.StatusCode.OK:
+            logging.error(
+                'abort() called with StatusCode.OK; returning UNKNOWN')
+            code = grpc.StatusCode.UNKNOWN
+            details = ''
```

Comment:
Tests are failing due to the need for details to be encoded with `_common.encode`

Reply:
Mmm. I noticed it gets encoded 3 lines below anyway. It's weird. Let me investigate further.